code/modules/shuttle/emergency.dm

```diff
@@ -292,6 +292,23 @@
 
 	return has_people && hijacker_present
 
+/obj/docking_port/mobile/emergency/proc/is_hijacked_by_xenos()
+	var/has_xenos = FALSE
+	for(var/mob/living/player in GLOB.alive_mob_list)
+		if(issilicon(player)) //Borgs are technically dead anyways
```

Comment:
Does normal hijack do this?

Reply:
consider scrolling about 30 lines upwards in this file :)